README.md

```diff
@@ -57,7 +57,8 @@ OFirehose require a configuration file located in `/etc/ofirehose.json`, `~/.ofi
    "/etc/ssl/private/myserver.key". Defaults `null`.
 * **cert** If you're using SSL, the path to the server cert, like
    "/etc/ssl/private/myserver.crt". Defaults `null`.
-* **address** The address to listen on. Defaults server hostname, Use this if you've got some kind of load-balancer or NAS or whatever and your local IP doesn't map to the IP of the hostname.
+* **server** The address on which "publish/subscribe"(`PubSubHubbub`) will happen. Defaults machine `hostname`.
```

Comment:
Nit: space between `"` and `(`

Nit: "Defaults to machine hostname" (with `to` and without code formatting)

Reply:
Also probably no code formatting on PubSubHubbub - it should probably say "PubSubHubbub/WebSub" and link to the spec (https://www.w3.org/TR/websub/)

Reply:
done